src/main/java/net/kaczmarzyk/spring/data/jpa/web/ProcessingContext.java

```diff
@@ -34,5 +33,5 @@ public interface ProcessingContext {
 
 	String getPathVariableValue(String pathVariableName);
 
-	BodyParams getBodyParams();
+	String[] getBodyParamValues(String bodyParamName);
```

Comment:
introduced new method returning body params as `String[]` to keep compatibility with the rest returning params methods